tools/db_crashtest.py

```diff
@@ -178,6 +178,7 @@
     "wal_compression": lambda: random.choice(["none", "zstd"]),
     "verify_sst_unique_id_in_manifest": 1,  # always do unique_id verification
     "secondary_cache_uri": "",
+    "allow_data_in_errors": True,
```

Comment:
So for automatic stress testing, we always log data after this PR?

Reply:
Right, with this being true, key, value, information is allowed to appear in LOG/status message.